libsplinter/Cargo.toml

```diff
@@ -120,6 +121,7 @@ biome-notifications = ["biome"]
 biome-user = ["biome"]
 circuit-template = ["glob"]
 events = ["actix-http", "futures", "hyper", "tokio", "awc"]
+oauth = []
```

Comment:
This should probably include `"auth"` in its list of features.

Reply:
Added